src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -220,4 +277,123 @@ private void saveModifications(NetworkStoreListener listener) {
     private Mono<Void> assertGroovyScriptNotEmpty(String groovyScript) {
         return StringUtils.isBlank(groovyScript) ? Mono.error(new NetworkModificationException(GROOVY_SCRIPT_EMPTY)) : Mono.empty();
     }
+
+    private void createLoadInNodeBreaker(VoltageLevel voltageLevel, LoadCreationInfos loadCreationInfos) {
+        // busId is a busbar section id
+        VoltageLevel.NodeBreakerView nodeBreakerView = voltageLevel.getNodeBreakerView();
+        BusbarSection busbarSection = nodeBreakerView.getBusbarSection(loadCreationInfos.getBusId());
+        if (busbarSection == null) {
+            throw new NetworkModificationException(BUSBAR_SECTION_NOT_FOUND, loadCreationInfos.getBusId());
+        }
+
+        // creating the disconnector
+        int newNode = nodeBreakerView.getMaximumNodeIndex();
+        nodeBreakerView.newSwitch()
+                .setId("disconnector_" + loadCreationInfos.getEquipmentId())
+                .setName("disconnector_" + loadCreationInfos.getEquipmentName())
```

Comment:
What about if `loadCreationInfos.getEquipmentName()` is not defined? I think we will end up with an ugly and useless name

Reply:
Done